frontend/test/utils/Implicits.scala

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import model.Eventbrite.{EBDescription, EBEvent, EventWithDescription}
+
+object Implicits {
+
+  implicit class Cheat(event: EBEvent) {
+    // this is just a way to get legacy events from the test data into the new format
+    // the proper way would be to recreate in eventbrite, and run the actual api calls to get it
+    def cheatyMigrate: EventWithDescription =
```

Comment:
I was a bit confused as to what this was doing because of the name - is there anything more descriptive we could use? `toEventWithSeparateDescription` (just off the top of my head)

Reply:
I wanted to make it obvious this was migrating the old events to the new format in the way I assumed would happen, rather than actually getting them from the API in the correct format (as the test data was produced years ago and setting it up again would be a big manual process)  However you're right I should have gone back and thought about the name again to be more clear, I'll go in and tweak it.